jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyManager.java

```diff
@@ -157,7 +157,7 @@ private static Optional<String> firstNonOptionArgument(String... lineComponents)
 
     // visible for testing
     static void verifyAssemblyReferencedInDockerfile(
-        File dockerFile, BuildConfiguration buildConfig, Properties properties, KitLogger log) throws IOException {
+        File dockerFile, BuildConfiguration buildConfig, AssemblyConfiguration assemblyConfig, Properties properties, KitLogger log) throws IOException {
         if (buildConfig.getAssembly() == null) {
             return;
         }
```

Comment:
I'm not sure if this should be changed too.
Also, I'm not sure if it might make sense to remove the BuildConfiguration parameter altogether